include/envoy/ssl/context_config.h

```diff
@@ -39,19 +39,34 @@ class ContextConfig {
   virtual const std::string& ecdhCurves() const PURE;
 
   /**
-   * @return The CA certificate file to use for peer validation.
+   * @return The CA certificate to use for peer validation.
    */
-  virtual const std::string& caCertFile() const PURE;
+  virtual const std::string& caCert() const PURE;
```

Comment:
Would it make more sense to return std::vector<uint8_t> or something?  Although protobuf uses std::string for non-string buffers, so we might be stuck with this.

Reply:
Maybe? I'm fine with either, but as you noted, protobuf uses `std::string` for the `bytes inline` field, so we might as well stick to it.

Reply:
Agreed; let's stick with what you had.